src/Razor/src/Microsoft.AspNetCore.Razor.Language/Intermediate/ComponentAttributeIntermediateNode.cs

```diff
@@ -76,6 +76,8 @@ public ComponentAttributeIntermediateNode(TagHelperPropertyIntermediateNode prop
 
         public BoundAttributeDescriptor BoundAttribute { get; set; }
 
+        public bool IsAttributeSplat { get; set; }
```

Comment:
I'd rather take an approach like `@bind` `@ref` etc. as shown in @ajaybhargavb's other PR: https://github.com/aspnet/AspNetCore-Tooling/pull/638

Reply:
Could you please be more specific?

Reply:
@NTaylorMullen I guess this still necessary because the code gen phase needs to call different methods based on this. Only way to not have this is to add a new IR node for splat.

Reply:
> Only way to not have this is to add a new IR node for splat.

That sounds cleaner to me.

Reply:
Addressed this in the top post. 

> If you have cool 😎 feedback about the IR representation, I'm open to hear ideas about what would be better. Adding a separate node type for AddMultipleAttributes would require a LOT of plumbing. To avoid putting a few ifs in a few places.

Reply:
Do y'all have any cool ideas? making this it's own node type is a serious amount of refactoring that I'm not sure is warrented.